ui/src/components/dashboard/audio_list_item.test.coffee

```diff
@@ -54,6 +54,25 @@ describe 'audio-list-item', ->
   it 'should not have an invisible icon', ->
     expect(@wrapper.find('.fa.fa-eye-slash')).to.have.length 0
 
+  describe 'list item accent color', ->
+    extension_check = (ext, color) =>
+      it 'should be '+color+' on '+ext, ->
```

Comment:
```coffee
it "should be #{color} on #{ext}" ->
```

Reply:
Yeah, good idea, I just personally like to stick to one either single quotes or double quotes :) Will do this though!